src/items.cpp

```diff
@@ -541,12 +541,7 @@ void Items::buildInventoryList()
 
 void Items::parseItemNode(const pugi::xml_node& itemNode, uint16_t id)
 {
-	if (id > 30000 && id < 30100) {
-		id -= 30000;
-
-		if (id >= items.size()) {
-			items.resize(id + 1);
-		}
+	if (id > 0 && id < 100) {
```

Comment:
id is unsigned 🤔

Reply:
so what?

Reply:
Sorry, had an impression that id 0 is considered invalid and filtered. This happens for  it.id just few lines below. Anyway, I wonder if moving it just after `it.id == 0` block wouldn't be correct? Then the `id > 0` would actually become redundant.